src/ui/actions/timeline.ts

```diff
@@ -126,8 +126,10 @@ export function jumpToInitialPausePoint(): UIThunkAction<Promise<void>> {
   return async (dispatch, getState, { replayClient }) => {
     const recordingId = getRecordingId(getState());
     assert(recordingId);
+
     const endpoint = await sessionEndPointCache.readAsync(replayClient);
-    dispatch(pointsReceived([endpoint]));
+    dispatch(setEndpoint(endpoint));
```

Comment:
The only place we cared about these points was here, and we only cared about the endpoint. We can just store that in Redux instead.